lib/blockchain.go

```diff
@@ -1691,6 +1691,9 @@ func (bc *Blockchain) processHeaderPoW(blockHeader *MsgDeSoHeader, headerHash *B
 		return false, false, HeaderErrorHeightInvalid
 	}
 
+	// Set ProposedInView to Height - this sets us up for PoS
+	blockHeader.ProposedInView = blockHeader.Height
+
```

Comment:
Not sure if we want to explicitly set the view on a PoW block here. Why not have a special case in processBlockPoS that checks if the parent block's height is < PoSCutoverForkHeight, and if so it just infers that the PoW block has its view = height?

This way, all rules to validate the PoS transition is coupled entirely within the processBlockPoS function